mediapipe/modules/face_geometry/geometry_pipeline_calculator_with_pose_output.cc

```diff
@@ -0,0 +1,183 @@
+#include <memory>
```

Comment:
Please, remove `mediapipe/modules/face_geometry/data/geometry_pipeline_metadata_landmarks.binarypb`, as I undestand `.binarypb` will be generated each time you run the face_mesh example, right?